bin/node/cli/src/cli.rs

```diff
@@ -117,3 +113,14 @@ pub enum Subcommand {
 	#[cfg(feature = "try-runtime")]
 	TryRuntime(try_runtime_cli::TryRuntimeCmd),
 }
+
+#[derive(Debug, StructOpt)]
+pub struct DynamicFeeParameters {
+	/// Maximum number of logs in a query.
+	#[structopt(long, default_value = "10000")]
+	pub max_past_logs: u32,
+
+	/// The dynamic-fee pallet target gas price set by block author
+	#[structopt(long, default_value = "1000000000")]
+	pub target_gas_price: u64,
+}
```

Comment:
Actually, `max_past_logs` has nothing to do with DynamicFee. I think to keep the old way would be better.

Reply:
Rename to `DvmArgs`.

Reply:
Still not push yet?